Model/Behavior/MongoBehavior.php

```diff
@@ -151,6 +156,26 @@ public function batchInsert(Model $model, array $data) {
 	}
 
 /**
+ * @param Model $model
+ * @param       $insert
+ */
+	public function addToBatch(Model $model, $insert){
+		$this->batchInsertArray[$model->alias][] = $insert;
+
+		if (sizeof($this->batchInsertArray[$model->alias]) > 1000){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of "sizeof" with "count". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2050)